### PR TITLE
[CI:DOCS] Update farm docs

### PR DESCRIPTION
**Pull request by umohnani8**
Add some updates and clarification to the
farm docs.

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: docs/source/markdown/podman-farm.1.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ Farm out builds to machines running Podman for different architectures.
 
 Manage farms by creating, updating, and removing them.
 
+Note: All farm machines must have an updated podman version of at least v4.9.0.
```
Comment (mheon) [inline docs/source/markdown/podman-farm.1.md:14]: Recommend replacing "updated podman version of at least" with "minimum Podman version of"

Comment (umohnani8) [reply docs/source/markdown/podman-farm.1.md]: Fixed

**Pull request merged**